Astro 10/25 release notes (#1383)
astronomer/docs: astro/view-clusters.md

In the Cloud UI, click the **Clusters** tab to view information about the clusters in your Organization. The following table provides descriptions for each cluster value.

| Value              | Description                                                                                                                 |
| ------------------ | --------------------------------------------------------------------------------------------------------------------------- |
| Name               | The name of the cluster                                                                                                     |

> | Provider           | The cloud provider that the cluster is hosted on                                                                            |

Reviewer: ```suggestion
| Provider           | The cloud provider that hosts the cluster                                                                            |
```

> | Account ID         | The ID of the cloud provider account that hosts your cluster. This is either an AWS account ID or a Google Cloud project ID |

Reviewer: ```suggestion
| Account ID         | The ID of the cloud provider account that hosts the cluster. This is either an AWS account ID or a Google Cloud project ID |
```

| Region             | The cloud provider region where the cluster is hosted                                                                       |
| VPC subnet         | The range of IP addresses that can be used to connect to the cluster's VPC                                                  |
| Pod subnet         | GCP only. The range of IP addresses that can be used to connect to the cluster's Pods                                       |
| Service subnet     | GCP only. The range of IP addresses that can be used to connect to the cluster's GCP services                               |
| Service peering    | GCP only. The range of IP addresses that can be used to peer with the cluster's VPC                                         |
| DB instance type   | The type of instance used for the cluster's primary database                                                                |
| Node instance type | The instance type used for the cluster's worker nodes                                                                       |
| Max node count     | The maximum number of worker nodes supported across all Deployments in the cluster                                          |

> | External IPs       | Public IP addresses for connecting the cluster to external services                                                         |

Author: New table item

Reviewer: Do you want to define VPC and GPC on first use?

> | External IPs       | Public IP addresses for connecting the cluster to external services                                                         |

Reviewer: ```suggestion
| External IPs       | The public IP addresses for connecting the cluster to external services                                                         |
```

| Updated            | The date and time the cluster's settings were last updated                                                                  |

> | Created            | The date and time at which the cluster was created                                                                          |

Reviewer: ```suggestion
| Created            | The date and time the cluster was created                                                                          |
```
